hyper/transport/grpc/suspend.go

```diff
@@ -0,0 +1,42 @@
+package grpc
+
+import (
+	"context"
+	"errors"
+
+	"github.com/isard-vdi/isard/common/pkg/grpc"
```

Comment:
This should be in the "external" import group (with "grpc/codes" and all the others)

Reply:
I don't catch this